examples/test/basic.test.js

```diff
@@ -1,27 +1,19 @@
-const test = require('ava')
-const axios = require('axios')
-const { server, app } = require('../basic/server')
+const test = require("ava");
```

Comment:
Let's not start holly war. If there were no style changes in the file let's revert this changes.

![](https://img.devrant.com/devrant/rant/r_154480_wNWJE.jpg)

Reply:
I just added `sendQuery` as shared `util/sendQuery.js` to shorten this test.